app/controllers/webapp/Profile.scala

```diff
@@ -247,31 +247,108 @@ class Profile @Inject() (
   }
   /**
    * NVM = non voting membership
-   * The controller will check if a user is in the posebility to be active member. 
-   * Store the user in a list for Network-ASP to check if the request is valid
+   * The controller will check if a user is in the possibility to be non voting member.
    */
 
-  /*def requestActiveFlag = SecuredAction.async { implicit request =>
-    userService.requestActiveFlag(request.identity).map(_ match {
-      Some()
-    })
-  }*/
+  def inActiveNVM = UserAwareAction.async { implicit request =>
+    request.identity match {
+      case Some(user) => {
+        user.profiles.headOption match {
+          case Some(profile) => userService.inActiveNVM(profile).map(status =>
+            WebAppResult.Ok(request, "profile.inactive.success", Nil, "Profile.inactive.success", Json.obj("status" -> "inactive")).getResult
```

Comment:
I think the `Result` (`WebAppResult.Ok` or something else, like `WebAppResult.InternalServerError`) should depend on the value of `status`. Nevertheless, it has to be part of the response. This is a nice-to-have.